test/parallel/test-https-client-get-url.js

```diff
@@ -31,7 +29,12 @@ var server = https.createServer(options, function(req, res) {
 });
 
 server.listen(common.PORT, function() {
-  https.get('https://127.0.0.1:' + common.PORT + '/foo?bar');
+  https.get({
+    host: '127.0.0.1',
+    port: common.PORT,
+    path: '/foo?bar',
+    rejectUnauthorized: false
+  });
 });
```

Comment:
> Maybe this test can pass an object to https.get(), and we can test URL parsing separately?

You might as well remove the test altogether in that case, its sole purpose is verifying that `https.get()` accepts a URL.